curl-wait-for-it.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+URL="$1"
+shift
+cmd="$@"
+
+counter=0
+
+until curl "$URL"; do
+  echo >&2 "$URL is unavailable - sleeping"
+  sleep 15
```

Comment:
Not necessary blocking the merge but this could be improved having a maximum number of retries to avoid having a script that never returns if something fails on the server setup

Reply:
@leomrocha fortunately dockerhub kills any containers trying to run more than 6 hours - so this one wouldn't be infinite, but yes, you are right, we should have a limit here... what do you think about 5 minutes? (`60 / 15 * 5 = 20` retries)